test/e2e/specs/plans/plans__legacy-renew.ts

```diff
@@ -44,12 +43,6 @@ describe( DataHelper.createSuiteTitle( 'Plans (Legacy): Renew' ), function () {
 		it( `${ planName } is the active plan`, async function () {
 			await plansPage.validateActivePlan( planTier );
 		} );
-
-		it( 'Legacy plan notice is shown', async function () {
-			const noticeComponent = new NoticeComponent( page );
-			const message = `You’re currently on a legacy plan. If you’d like to learn about your eligibility to switch to a Pro plan please contact support.`;
-			await noticeComponent.noticeShown( message );
-		} );
```

Comment:
I think I saw a similar changeset on another one of the revert PRs. Might want to make sure this is actually necessary.

Reply:
Thanks for letting me know @worldomonation, I'll rebase and double check it